Tiltfile

```diff
@@ -1,55 +1,132 @@
 for resource in manager_yaml:
     if resource["metadata"]["name"] == "capl-manager-credentials":
-        resource["stringData"]["apiToken"] = os.getenv('LINODE_TOKEN')
-    if resource["kind"] == "CustomResourceDefinition" and resource["spec"]["group"] == "infrastructure.cluster.x-k8s.io":    
+        resource["stringData"]["apiToken"] = os.getenv("LINODE_TOKEN")
+    if resource["kind"] == "CustomResourceDefinition" and resource["spec"]["group"] == "infrastructure.cluster.x-k8s.io":
         resource["metadata"]["labels"]["clusterctl.cluster.x-k8s.io"] = ""
 k8s_yaml(encode_yaml_stream(manager_yaml))
 
+docker_build(
+    "docker.io/linode/cluster-api-provider-linode",
+    context=".",
+    only=("Dockerfile", "Makefile", "vendor", "go.mod", "go.sum",
+    "./api", "./cloud", "./cmd", "./controller", "./util", "./version",),
+    build_args={"VERSION": os.getenv("VERSION", "")},
+)
+
 k8s_resource(
     workload="capl-controller-manager",
     objects=[
-       "capl-system:namespace",
-       "linodeclusters.infrastructure.cluster.x-k8s.io:customresourcedefinition",
-       "linodemachines.infrastructure.cluster.x-k8s.io:customresourcedefinition",
-       "linodeclustertemplates.infrastructure.cluster.x-k8s.io:customresourcedefinition",
-       "linodemachinetemplates.infrastructure.cluster.x-k8s.io:customresourcedefinition",
-       "linodevpcs.infrastructure.cluster.x-k8s.io:customresourcedefinition",
-       "linodeobjectstoragebuckets.infrastructure.cluster.x-k8s.io:customresourcedefinition",
-       "capl-controller-manager:serviceaccount",
-       "capl-leader-election-role:role",
-       "capl-manager-role:clusterrole",
-       "capl-metrics-reader:clusterrole",
-       "capl-proxy-role:clusterrole",
-       "capl-leader-election-rolebinding:rolebinding",
-       "capl-manager-rolebinding:clusterrolebinding",
-       "capl-proxy-rolebinding:clusterrolebinding",
-       "capl-manager-credentials:secret",
-   ]
+        "capl-system:namespace",
+        "linodeclusters.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+        "linodemachines.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+        "linodeclustertemplates.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+        "linodemachinetemplates.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+        "linodevpcs.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+        "linodeobjectstoragebuckets.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+        "capl-controller-manager:serviceaccount",
+        "capl-leader-election-role:role",
+        "capl-manager-role:clusterrole",
+        "capl-metrics-reader:clusterrole",
+        "capl-proxy-role:clusterrole",
+        "capl-leader-election-rolebinding:rolebinding",
+        "capl-manager-rolebinding:clusterrolebinding",
+        "capl-proxy-rolebinding:clusterrolebinding",
+        "capl-manager-credentials:secret",
+    ],
+    labels=["CAPL"],
```

Comment:
I think we should add a dependency here to capi controller manager. I noticed that CAPL gets installed first before cert-manager and CAPI CRDs which is causing it to throw errors